src/config/dynamic-theme-fixes.config

```diff
@@ -2842,6 +2842,15 @@ CSS
 
 ================================
 
+caf.fr
+
+CSS
+#theme-contenu-cnaf .row[class*="conteneur-"][class*="-cnaf"] > [class*="col-"], #theme-contenu-menu-background-cnaf {
```

Comment:
```suggestion
#theme-contenu-cnaf .row[class*="conteneur-"][class*="-cnaf"] > [class*="col-"],
#theme-contenu-menu-background-cnaf {
```

Reply:
Hi, I did fix the issue in amended commit pushed right now.